webapp/channels/src/utils/message_html_to_component.tsx

```diff
@@ -29,25 +57,25 @@ import PostEditedIndicator from 'components/post_view/post_edited_indicator';
  * - hasPluginTooltips - If specified, the LinkTooltip component is placed inside links. Defaults to false.
  * - channelId = If specified, to be passed along to ProfilePopover via AtMention
  */
-export function messageHtmlToComponent(html, options = {}) {
+export function messageHtmlToComponent(html: any, options: Options = {}) {
```

Comment:
It would seem that `html` is always a string. Why any?

Reply:
Yes, after digging, looks like html is always a string, the problem was the types of react-to-html was putting `any` there.